use-cases/multiple-accounts-k8s-threat.md

```diff
@@ -1,229 +1,238 @@
-
-Helm Cloud-Connector chart requires specific AWS credentials to be passed by parameter, a new user + access key will
-be created within account, to be able to fetch the events in the S3 bucket (1) or several S3 buckets (2)
-<br/><br/>
-WIP.
-<br/><br/>
-We currently provide a module to create first use-case,
-[`modules/infrastructure/permissions/iam-user`](https://github.com/sysdiglabs/terraform-aws-secure-for-cloud/blob/master/modules/infrastructure/permissions/iam-user) (1),
-but can work on providing a way to automatize the later (2)
-
-<!--
-```terraform
-module "multi-account" {
-   providers = {
-      aws = aws.s3
-   }
-   source  = "sysdiglabs/secure-for-cloud/aws//modules/infrastructure/permissions/iam-user"
-   deploy_image_scanning         = false
-   cloudtrail_s3_bucket_arn      = module.cloudtrail_s3_sns_sqs.cloudtrail_s3_arn
-   cloudtrail_subscribed_sqs_arn = module.cloudtrail_s3_sns_sqs.cloudtrail_subscribed_sqs_arn
-}
-```
--->
-
-5. **Sysdig workload deployment on K8s**
+5. Deploy the Sysdig workload on Kubernetes:
 
    * Populate  `sysdig_secure_url`, `SYSDID_SECURE_API_TOKEN` and `REGION`
-   * WIP. enable terraform module to be able to define [`nodeSelector` and `tolerations` parameters of the cloud-connector helm chart](https://charts.sysdig.com/charts/cloud-connector/#configuration)
-
-```terraform
-resource "helm_release" "cloud_connector" {
-
-  provider = helm
-
-  name = "cloud-connector"
-
-  repository = "https://charts.sysdig.com"
-  chart      = "cloud-connector"
-
-  create_namespace = true
-  namespace        = "sysdig"
-
-  set {
-    name  = "image.pullPolicy"
-    value = "Always"
-  }
-
-  set {
-    name  = "sysdig.url"
-    value =  "<sysdig_secure_url>"
-  }
-
-  set_sensitive {
-    name  = "sysdig.secureAPIToken"
-    value = "<SYSDIG_SECURE_API_TOKEN>"
-  }
-
-  set_sensitive {
-    name  = "aws.accessKeyId"
-    value = "<AWS_ACCESS_KEY>"
-  }
-
-  set_sensitive {
-    name  = "aws.secretAccessKey"
-    value = "<AWS_SECRET_ACCESS_KEY>"
-  }
-
-  set {
-    name  = "aws.region"
-    value = "<REGION>"
-  }
-
-  set {
-    name  = "nodeSelector.<NODE_SELECTOR_LABEL>"
-    value = "<NODE_SELECTOR_LABEL_VALUE>"
-  }
-
-  set {
-    name  = "tolerations[0].key"
-    value = "<TOLERATION_KEY>"
-  }
-
-  set {
-    name  = "tolerations[0].operator"
-    value = "<TOLERATION_OPERATOR>"
-  }
-
-  set {
-    name  = "tolerations[0].value"
-    value = "<TOLERATION_VALUE>"
-  }
-
-  set {
-    name  = "tolerations[0].effect"
-    value = "<TOLERATION_EFFECT>"
-  }
-
-  values = [
-    <<CONFIG
-logging: info
-ingestors:
-  - aws-cloudtrail-s3-sns-sqs:
-      queueURL: CLOUDTRAIL_S3_SNS_SQS_URL
-CONFIG
-  ]
-}
-
-```
+
+   * WIP. Enable terraform module to be able to define [`nodeSelector` and `tolerations` parameters of the cloud-connector helm chart](https://charts.sysdig.com/charts/cloud-connector/#configuration)
+
+     ```json
```

Comment:
careful with the extra spaces/trailing in the begining of the lines, since it must be a valid json.